cohortextractor/cohortextractor.py

```diff
@@ -685,6 +686,10 @@ def main():
         help="Do not regenerate data if output file already exists",
         action="store_true",
     )
+    generate_cohort_parser.add_argument(
+        "--with-end-date-fix",
+        action="store_true",
```

Comment:
I'd be tempted to reverse the logic here. Don't we want it fixed by default, and then separately we want to be able to re-enable the bug to do some investigation as to its impact. Is there value to keeping the bug in place?

Reply:
I had a comment explaining this in the missing flags.py.  I've moved that comment into cohortextractor.py -- what do you think?

Reply:
Yes, that seems reasonable